service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandlerTest.java

```diff
@@ -176,6 +174,17 @@
     @InjectMocks
     private NotificationHandler notificationHandler;
 
+    @BeforeEach
+    void setup() {
+        notificationHandler = new NotificationHandler(hmctsCourtLookupConfiguration,
```

Comment:
now  @InjectMocks on private NotificationHandler notificationHandler;
does not make sense, as you dont use DI here